src/Service/RequestHandler/GetV2.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Copyright Shopgate Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @author    Shopgate Inc, 804 Congress Ave, Austin, Texas 78701 <interfaces@shopgate.com>
+ * @copyright Shopgate Inc
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License, Version 2.0
+ */
+
+namespace Shopgate\CloudIntegrationSdk\Service\RequestHandler;
+
+use Shopgate\CloudIntegrationSdk\Service\Authenticator;
+use Shopgate\CloudIntegrationSdk\Repository;
+
+use Shopgate\CloudIntegrationSdk\ValueObject\Request;
+use Shopgate\CloudIntegrationSdk\ValueObject\Response;
+
```

Comment:
Do we want to group the uses with lines or just the optimize imports feature of PHPStorm (automatically sorts the imports by name ascending)?

Reply:
I guess that's up for discussion. I like it a bit better, when they are sorted by the way, how they appear in the project directory. I switched the first two ones to match that. I removed tha blank line inbetween, though.